relay-redis/src/real.rs

```diff
@@ -382,3 +396,119 @@ pub struct Stats {
     /// The number of idle connections.
     pub idle_connections: u32,
 }
+
+/// [`bb8::ManageConnection`] for a async redis cluster.
+pub struct RedisClusterConnectionManager {
+    client: ClusterClient,
+}
+
+impl RedisClusterConnectionManager {
+    /// Creates a new [`bb8::ManageConnection`] with the provided [`ClusterClient`]
+    pub fn new(client: ClusterClient) -> Self {
+        Self { client }
+    }
+}
+
+impl bb8::ManageConnection for RedisClusterConnectionManager {
```

Comment:
This would make a nice contribution to `bb8`, if they're inclined to accept it.

Reply:
Good idea, I will check if they are up for it